recipes/quantization.md

```diff
@@ -1,13 +1,43 @@
 # Quantization and Sparsity
 
-torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) for more details.
+torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
 
 #### Quantize
 To quantize a model (default is int4 weight only quantization):
 ```
 tune run quantize --config quantization
 ```
 
+#### Quantization-Aware Training (QAT)
+
+(PyTorch 2.4+)
+
+QAT refers to applying fake quantization to weights and/or activations during finetuning,
+which means simulating only the quantization math without actually casting the original
+dtype to a lower precision.
+
+```
+tune run qat_distributed --config qat
```

Comment:
Need nproc_per_node here? Also nit: Add some segue before the code block, e.g. "QAT can be run via the command"